src/shader/shaderLightSampling.js

```diff
@@ -73,14 +110,103 @@ LightSampleRec randomRectAreaLightSample( Light light, vec3 rayOrigin ) {
 
 }
 
+LightSampleRec randomCircularAreaLightSample( Light light, vec3 rayOrigin ) {
+
+	LightSampleRec lightSampleRec;
+	lightSampleRec.hit = true;
+
+	lightSampleRec.emission = light.color * light.intensity;
+
+	float r = 2.0 * sqrt( rand() );
```

Comment:
I think this should be `0.5 * sqrt( rand() )` since theta is sampled from the full `2 * PI` range and the width / height represent the full width and full height of the light, not just the radius size.

Reply:
Checked this, it resolves both the light bleed and MIS/non-MIS discrepancy. Indeed the width/height representation requires we are using [-0.5, 0.5].